cmd/cerbos/compile/internal/colored/colored.go

```diff
@@ -6,15 +6,21 @@ package colored
 import "github.com/fatih/color"
 
 var (
-	Header         = color.New(color.FgHiWhite, color.Bold).SprintFunc()
-	FileName       = color.New(color.FgHiCyan).SprintFunc()
-	ErrorMsg       = color.New(color.FgHiRed).SprintFunc()
-	SkippedTest    = color.New(color.FgHiWhite).SprintFunc()
-	FailedTest     = color.New(color.FgHiRed).SprintFunc()
-	SuccessfulTest = color.New(color.FgHiGreen).SprintFunc()
-	Suite          = color.New(color.FgHiBlue, color.Bold).SprintFunc()
-	Principal      = color.New(color.FgHiCyan).SprintFunc()
-	Resource       = color.New(color.FgHiBlue).SprintFunc()
-	Action         = color.New(color.FgYellow).SprintFunc()
-	Trace          = color.New(color.FgHiWhite).SprintFunc()
+	Header                  = color.New(color.FgHiWhite, color.Bold).SprintFunc()
+	FileName                = color.New(color.FgCyan).SprintFunc()
+	ErrorMsg                = color.New(color.FgRed).SprintFunc()
+	SkippedTest             = color.New(color.FgHiWhite).SprintFunc()
+	FailedTest              = color.New(color.FgRed).SprintFunc()
+	SuccessfulTest          = color.New(color.FgGreen).SprintFunc()
+	Suite                   = color.New(color.FgBlue, color.Bold).SprintFunc()
+	Principal               = color.New(color.FgCyan).SprintFunc()
+	Resource                = color.New(color.FgBlue).SprintFunc()
+	Action                  = color.New(color.FgYellow).SprintFunc()
```

Comment:
I felt like the high-intensity colours were a bit overwhelming, so I dropped them down to regular intensity.